connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -106,12 +106,20 @@ public class EmbeddedKafkaCluster {
     private final String[] currentBrokerLogDirs;
     private final boolean hasListenerConfig;
 
+    final Map<String, String> additionalClientConfigs;
```

Comment:
What about using `clientConfigs`? If you agree let's rename the setters too.

Reply:
done